<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="bClearDetection">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>530</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear detection</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bSelect">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>350</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bUnselect">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Unselect All</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bRestart">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>restart</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinRes">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinThresholdZone">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Resolution</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>275</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Threshold
zone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bShutdown">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Shutdown</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinThreshold">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinExposition">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
    <property name="value">
     <number>175</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Exposition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSave">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>save zones</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bLoad">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>load zones</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinHeight">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2048</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>240</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinWidth">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2048</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>320</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSnap">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>snap</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioEnabled">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>enabled</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bAutoLearn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>start</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Auto learn</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
